code/modules/mob/living/carbon/xenomorph/attack_alien.dm

```diff
@@ -194,6 +194,8 @@
 	if (X.fortify)
 		return FALSE
 
+	SEND_SIGNAL(X, COMSIG_XENOMORPH_ATTACK_LIVING, src, damage_amount, X.xeno_caste.melee_damage * X.xeno_melee_damage_modifier)
```

Comment:
Fun fact, this might make ravagers actually able to heal from slashing other xenos

Reply:
Though, I should probably make sure this only applies for enemy xenos

Reply:
Had to redo this part as a result, as my hypothesis was proved correct